src/clr-angular/popover/dropdown/dropdown-trigger.spec.ts

```diff
@@ -49,5 +52,25 @@ export default function(): void {
     it('declares itself to the DropdownFocusHandler', function(this: Context) {
       expect(this.getClarityProvider(DropdownFocusHandler).trigger).toBe(this.clarityElement);
     });
+
+    it('aria label should receive the correct value', function(this: Context) {
+      const ifOpenService = this.getProvider(IfOpenService);
+
+      ifOpenService.open = false;
+      this.detectChanges();
+      expect(this.clarityElement.getAttribute('aria-label')).toBe('Open dropdown');
+
+      this.hostComponent.openLabel = 'New value for open';
+      this.detectChanges();
+      expect(this.clarityElement.getAttribute('aria-label')).toBe('New value for open');
+
+      ifOpenService.open = true;
+      this.detectChanges();
+      expect(this.clarityElement.getAttribute('aria-label')).toBe('Close dropdown');
```

Comment:
Should grab the common strings service and test against the default value instead of hard coding it here inside of toBe, same for next expect.